File: Python/c01_begginer/n1012/area.py
PI = 3.14159

data = input()

A, B, C = map(float, data.split())

triangle_area = (A * C) / 2
circle_area = PI * (C ** 2)
trapezium_area = ((A + B) * C) / 2
square_area = B ** 2
rectangle_area = A * B

print(
    f"TRIANGULO: {triangle_area:.3f}\n"
    f"CIRCULO: {circle_area:.3f}\n"
    f"TRAPEZIO: {trapezium_area:.3f}\n"
    f"QUADRADO: {square_area:.3f}\n"
    f"RETANGULO: {rectangle_area:.3f}"
)

File: JavaScript/c01_begginer/n1012/area.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

const [A, B, C] = lines.shift().split(' ').map(Number);

const PI = 3.14159;

const triangleArea = (A * C) / 2;
const circleArea = PI * (C ** 2);
const trapeziumArea = ((A + B) * C) / 2;
const squareArea = B ** 2;
const rectangleArea = A * B;

console.log(
  `TRIANGULO: ${triangleArea.toFixed(3)}\n` +
  `CIRCULO: ${circleArea.toFixed(3)}\n` +
  `TRAPEZIO: ${trapeziumArea.toFixed(3)}\n` +
  `QUADRADO: ${squareArea.toFixed(3)}\n` +
  `RETANGULO: ${rectangleArea.toFixed(3)}`
);
